ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/CeilKudf.java

```diff
@@ -14,16 +14,42 @@
 
 package io.confluent.ksql.function.udf.math;
 
-import io.confluent.ksql.function.KsqlFunctionException;
-import io.confluent.ksql.function.udf.Kudf;
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
 
-public class CeilKudf implements Kudf {
+@UdfDescription(name = CeilKudf.NAME, version = "Confluent",
+    description = "Returns the ceiling of a value.")
+public class CeilKudf {
+  static final String NAME = "CEIL";
 
-  @Override
-  public Object evaluate(final Object... args) {
-    if (args.length != 1) {
-      throw new KsqlFunctionException("Ceil udf should have one input argument.");
+  @Udf(description = "Returns the ceiling of a value.")
+  public Integer ceil(@UdfParameter(description = "A value.")
```

Comment:
nit: can we have the param and its annotations on the same line please (Same for other UDF methods):

```java
public Integer ceil(
   @UdfParameter(description = "A value.") final Integer n
) {
```

Also, the description `A value` isn't of much _value_ ;). 

This is displayed in the help for the function. I'd take a second look at the documentation you're adding in the UDF annotations in this PR. Run `DESCRIBE FUNCTION ceil` to see the output! 

Maybe a better description, i.e. one that would read better, would be `INT value`.